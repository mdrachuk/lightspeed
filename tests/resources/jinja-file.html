<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>{{ source.file }}</title>
</head>
<body>
file: {{ source.file }}
file.name: {{ source.file.name }}
file.extension: {{ source.file.extension }}

</body>
</html>