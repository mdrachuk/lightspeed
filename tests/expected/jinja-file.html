<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>jinja-file.html</title>
</head>
<body>
file: jinja-file.html
file.name: jinja-file
file.extension: html

</body>
</html>